docs/models/migrations.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 12
+---
+
+# Migrations
+
+Migrations are the recommended way to make changes to your database schema in production.
+They are a way to version control your database schema, allowing you to easily upgrade and downgrade your database as your application evolves.
```

Comment:
```suggestion
They are ways to version control your database schema, allowing you to easily upgrade and downgrade your database as your application evolves.
```

Or: there is a way - not sure though.

Reply:
not there are, they are, the current phrasing looks correct to me